### PR TITLE
[PYTHON] - Create Binary Search

### PR DESCRIPTION
**Pull request by tinyCodersDen**
Solution to code for implementing Binary Search in Python.

### CODE REVIEW

File: Python/Binary Search.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    
+    if elements[guess_index]<target:
+        min_index=guess_index
+        BinarySearch(guess_index,max_index)
+    
+    elif elements[guess_index]>target:
+        max_index=guess_index
+        BinarySearch(min_index,guess_index)
+
+BinarySearch(0,len(elements)-1)
```
Comment (its0din-ai) [inline Python/Binary Search.py]: Why the length of elements is subtracted by 1?
Because if the length is subtracted by 1. we cannot find 101 as the target

**Review by its0din-ai — CHANGES_REQUESTED** [review]
remove the array length subtraction
and set the target to 101 (last index)

**Comment by its0din-ai** [comment]
And catch the Error if target isn't present on the array

**Comment by its0din-ai** [comment]
We will take over the code at 10 AM (GMT)

**Review by dvoraksz — COMMENTED** [review]
Catch the error

**Review by its0din-ai — APPROVED** [review]

**Comment by its0din-ai** [comment]
**THANKS FOR PARTICIPATING**

**Pull request merged**